### ISSUE TITLE
No ships in hangar 

### ISSUE THREAD

**Issue by Kripto-hub** [onikamigames/starshipgame#186]
I have tried repeatedly to clear my connections and even reload the MM and Anchor walloet apps but I still cannot get ships to appear in my hangar and also still can't claim the initial kyanite. I can see my nfts and get "already synched" and player account not found errors. 
Windows 
 My MM is   0xEdFD4DC995830F83A7C874B7A9E4247Ca01174D7
My wax testnet is togatoga4321
browser is latest chrome Version 96.0.4664.110 (Official Build) (64-bit)

currently have 18 starship tokens value $16 today

Have tried to load and try on a different PC with same result no ships 

![noships1](https://user-images.githubusercontent.com/96148400/146655951-33fd8568-4d95-48a9-9cb7-fcbe27f47bcf.JPG)

![togatoga4321](https://user-images.githubusercontent.com/96148400/146655952-a1ad88eb-8fb8-4293-8c01-e9ecb3c82002.JPG)
![metamask1](https://user-images.githubusercontent.com/96148400/146655955-9aea517b-b810-403f-b25b-65607009f326.JPG)
![cantcollect1](https://user-images.githubusercontent.com/96148400/146655959-facaaabe-18cd-4cdc-ab4a-66cebc64e97f.JPG)

**Comment by Kripto-hub** [comment]
really acts like its not seeing my MM token info 
![image](https://user-images.githubusercontent.com/96148400/146657402-dbae85a1-e0e8-4140-a038-f42fef5b6135.png)

**Comment by Kripto-hub** [comment]
Found the issue with a little help from Discord.  
SO here is the problem, when I connected my MM to this thing the first time, I had an existing MM connection to my live Wax Wallet io account.  I had done this long ago when I had transferred my wax tokens for the pack sales.  When I connected to the testnet game the first time here it used this existing connection when I joined MM and then totally ignored the choice I made in the later step for my testnet account.  
I therefore need someone from the Dev team to delete the user 4t3yg.wam which is my live wax wallet.  I cannot join the game again with my MM and the right wallet until this existing account is gone no matter how many times I delete the connections in MM. Once it's in there,  I guess you are stuck with that until a dev can clear it. 
Don't know if there is a way when the game goes live to ever change a user detail or delete it yourself so its an extra word of caution to take or you are stuck with it. 

Just scanning the existing user list there are a few more live wallets listed in here that are probably others with the same issue. 
Here is the line I found on the contract list:
 https://wax-test.bloks.io/account/starshipnfts?tab=Tables&account=starshipnfts&scope=starshipnfts&limit=100&loadContract=true&table=players

13 | 4t3yg.wam | 0.00000000000000000 | ......1ckvad1 | 0xedfd4dc995830f83a7c874b7a9e4247ca01174d7 | 18.31004524230957031 | [ 861, 862, 863 ] | 0
